### PR TITLE
tidb-in-kubernetes: support force upgrade when pd cluster is unavailable

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
Support force upgrade when pd cluster is unavailable

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1799
https://github.com/pingcap/docs-cn/pull/1930

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/upgrade/tidb-cluster.md [hunk 1]
```diff
@@ -58,3 +58,25 @@ You can change the configuration of TiDB cluster through the following steps:
 >
 > - Changing the `enableConfigMapRollout` variable against a running cluster will trigger a rolling update of PD, TiKV, TiDB servers even if there's no change to the configuration.
 > - Currently, PD's `scheduler` and `replication` configurations (the configuration key under `[scheduler]` and `[replication]` of the PD configuration file) can not be upgraded automatically.  You must upgrade them manually via `pd-ctl`. See [pd-ctl](/dev/reference/tools/pd-control.md) for reference.
+
+## Force an upgrade of TiDB cluster
+
+If the PD cluster is unavailable due to factors such as PD configuration error, PD image tag error and NodeAffinity, then [scaling the TiDB cluster](/dev/tidb-in-kubernetes/scale-in-kubernetes.md), [upgrading the TiDB cluster](#upgrade-the-version-of-tidb-cluster) and [changing the TiDB cluster configuration](#change-the-configuration-of-tidb-cluster) cannot be operated. In this case, you can use `force-upgrade` (the version of TiDB Operator must be later than v1.0.0-beta.3) to force an upgrade of the cluster to recover cluster functionality. The specific steps are as follows:
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/upgrade/tidb-cluster.md]: "PD image tag error and NodeAffinity, etc."

Comment (anotherrachel) [reply dev/tidb-in-kubernetes/upgrade/tidb-cluster.md]: Thanks for pointing out. I rephrased it by using "such as", which is more suitable here. Is there anything else?

Comment (DanielZhangQD) [reply dev/tidb-in-kubernetes/upgrade/tidb-cluster.md]: No, thanks!

File: dev/tidb-in-kubernetes/upgrade/tidb-cluster.md [hunk 2]
```diff
@@ -58,3 +58,25 @@ You can change the configuration of TiDB cluster through the following steps:
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl annotate --overwrite tc <release-name> -n <namespace> tidb.pingcap.com/force-upgrade=true
+```
+
+Then execute `helm upgrade` command from the corresponding operation.
```
[the remark this answers is not in the document]
Comment (DanielZhangQD) [reply dev/tidb-in-kubernetes/upgrade/tidb-cluster.md]: It means the operations of [scaling the TiDB cluster](/dev/tidb-in-kubernetes/scale-in-kubernetes.md), [upgrading the TiDB cluster](#upgrade-the-version-of-tidb-cluster) and [changing the TiDB cluster configuration](#change-the-configuration-of-tidb-cluster).

[the remark this answers is not in the document]
Comment (yikeke) [reply dev/tidb-in-kubernetes/upgrade/tidb-cluster.md]: Ok, I get it. To clarify on this, please refer to the changes in https://github.com/pingcap/docs-cn/pull/1930, and update this PR after https://github.com/pingcap/docs-cn/pull/1930 is merged. Thanks~ @anotherrachel

[the remark this answers is not in the document]
Comment (anotherrachel) [reply dev/tidb-in-kubernetes/upgrade/tidb-cluster.md]: > Ok, I get it. To clarify on this, please refer to the changes in [pingcap/docs-cn#1930](https://github.com/pingcap/docs-cn/pull/1930), and update this PR after [pingcap/docs-cn#1930](https://github.com/pingcap/docs-cn/pull/1930) is merged. Thanks~ @anotherrachel

Copy that~

**Comment by anotherrachel** [comment]
@yikeke  Comments addressed, PTAL again~

**Pull request merged**